Week1/Newton_sqroot/Sqroot_Newton.cpp

```diff
@@ -3,11 +3,11 @@
 
 double sqroot_Newton(double x) {
 	double eps = 0.00000000001;
-	double cur = x;
+	double cur = 1;
 	double next = x;
 
 	while (true) {
```

Comment:
why not put if condition into the while? The only thing that would be sacrificed is the one extra assignment, which could be optimized by compiler further,